codelabs/background_segmenter/code/index.js

```diff
@@ -0,0 +1,342 @@
+// we start by importing the tasks-vision module from the CDN
+import vision from "https://cdn.skypack.dev/@mediapipe/tasks-vision";
```

Comment:
Use NPM or use jsdelivr in the HTML. This is the best option for showing the package in Codepen, but it's not really what we want people to use in the wild